rasa/train.py

```diff
@@ -125,10 +132,32 @@ async def handle_domain_if_not_exists(
     return nlu_model_only
 
 
+def dry_run_result(fingerprint_comparison: FingerprintComparisonResult) -> Tuple[int, Text]:
+    """Returns a result of fingerprint comparison.
+
+    Args:
+        fingerprint_comparison: A result of fingerprint comparison operation.
+
+    Returns:
+        A tuple of code and human-readable text.
+    """
+    if not fingerprint_comparison.nlu and not fingerprint_comparison.core:
```

Comment:
There is another scenario where responses are changed in the domain, so the domain will be updated but there will be no training (i.e. just `should_retrain_nlg`). I wonder if it's worth handling this case too?

Reply:
Yes, we've been talking about it in another PR already. Gonna be fixed.